BankManager.swift

```diff
@@ -19,17 +19,55 @@ struct BankManager {
     }
     
     mutating func performTask(customer: Customer, semaphore: DispatchSemaphore) {
-        let second: Double = 1_000_000
         self.state = .working
-        print("\(BankManangerMessage.start)".format(customer.waitNumber, customer.priority.description, customer.taskType.rawValue))
-        usleep(useconds_t(customer.taskType.timeForTask * second))
-        print("\(BankManangerMessage.end)".format(customer.waitNumber, customer.priority.description, customer.taskType.rawValue))
+        switch customer.taskType {
+        case .loan:
+            checkLoanDocument(customer: customer)
+            requestLoanApprovalToHeadquarter(customer: customer)
+            implementLoanTask(customer: customer)
+        default:
+            implementDepositTask(customer: customer)
+        }
         self.state = .notWorking
         semaphore.signal()
     }
     
+    private func checkLoanDocument(customer: Customer) {
+        print("\(BankManagerMessage.start)".format(customer.waitNumber, customer.priority.description, customer.taskType.rawValue))
```

Comment:
제 생각에는 
```suggestion
        print(BankManagerMessage.start.format(customer.waitNumber, customer.priority.description, customer.taskType.rawValue))
```

이렇게 가능할 것 같은데 안되나요?